com.vargary.focal.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+
+Name=Focal
+Categories=Utility;
+
+Icon=com.vargary.focal
+Exec=/app/focal
```

Comment:
this should NOT be a full path. This should just be the name of the executable and the executable, of course, should be in the path, ie in `/app/bin`